tests/local_mode/test_doc.py

```diff
@@ -82,16 +82,52 @@ def test_avg_vector_non_valid():
     assert (actual == expected_vector).all()
 
 
-def test_add_custom_attr_doc():
-    """Test adding custom attribute to Doc objects"""
+def test_add_custom_attr():
+    """Test adding custom attributes to Doc & Token objects"""
 
     doc = nlp("Joey doesnt share food")
+    token = doc[0]
 
-    # add new custom attribute
-    doc.set_attribute(name="my_custom_tag", value="tag")
+    # add new custom attributes
+    doc.set_attribute(name="doc_tag", value="doc_value")
+    token.set_attribute(name="token_tag", value="token_value")
+
+    # check if custom attributes have been added
+    assert hasattr(doc._, "doc_tag") and doc._.doc_tag == "doc_value"
+    assert hasattr(token._, "token_tag") and token._.token_tag == "token_value"
+
+
+def test_check_custom_attr():
+    """Test if Doc and Token custom attributes exist"""
+
+    doc = nlp("Joey doesnt share food")
+    token = doc[0]
+
+    # add new custom attributes
+    doc.set_attribute(name="doc_tag", value="doc_value")
+    token.set_attribute(name="token_tag", value="token_value")
+
+    # check if they exists
+    assert doc.has_attribute("doc_tag")
+    assert token.has_attribute("token_tag")
 
-    # check custom attribute has been added
-    assert hasattr(doc._, "my_custom_tag") and doc._.my_custom_tag == "tag"
+
+def test_remove_custom_attr():
+    """Test the deletion of a custom Doc attribute"""
+
+    doc = nlp("Joey doesnt share food")
+    token = doc[0]
+
+    # add new custom attributes
+    doc.set_attribute(name="doc_tag", value="doc_value")
+    token.set_attribute(name="token_tag", value="token_value")
+
+    # remove them
```

Comment:
it will be much better, if you type the entire thing, "them" - > "new custom attributes" same below also